crates/solidity/testing/snapshots/bindings_output/expressions/type_expr/generated/0.4.11.txt

```diff
@@ -0,0 +1,29 @@
+# This file is generated automatically by infrastructure scripts. Please don't edit by hand.
+
+References and definitions: 
+   ╭─[crates/solidity/testing/snapshots/bindings_output/expressions/type_expr/input.sol:1:1]
+   │
+ 1 │ contract Test {
+   │          ──┬─  
+   │            ╰─── def: 1
+ 2 │     enum Answer { Yes, No }
+   │          ───┬──   ─┬─  ─┬  
+   │             ╰────────────── def: 2
+   │                    │    │  
+   │                    ╰─────── def: 3
+   │                         │  
+   │                         ╰── def: 4
+ 3 │     function testFunc() {
+   │              ────┬───  
+   │                  ╰───── def: 5
+ 4 │         string name = type(Test).name;
+   │                ──┬─  
+   │                  ╰─── def: 6
+ 5 │         Answer first = type(Answer).min;
+   │         ───┬── ──┬──  
+   │            ╰────────── ref: 2
+   │                  │    
+   │                  ╰──── def: 7
+   │ 
+   │ Note: WARNING: Parsing failed. Results may be incomplete.
```

Comment:
Same suggestion here:
I suggest printing the errors directly instead of this warning, so that readers can correlate any failed bindings to the parse errors directly.

Maybe another section below?

```log
Parse Errors:

   ╭─[crates/solidity/testing/snapshots/bindings_output/expressions/type_expr/input.sol:1:1]
   │
     ....... etc...
```

Reply:
:heavy_check_mark: